inc/ticket.class.php

```diff
@@ -140,11 +128,16 @@ static function getConsumedForCreditEntity($ID) {
       global $DB;
 
       $tot   = 0;
-      $query = "SELECT SUM(`consumed`)
-                FROM `".self::getTable()."`
-                WHERE `plugin_credit_entities_id` = '".$ID."'";
 
-      if ($result = $DB->query($query)) {
+      $request = [
+         'SELECT' => ['SUM' => 'consumed'],
+         'FROM'   => self::getTable(),
+         'WHERE'  => [
+            'plugin_credit_entities_id' => $ID
+         ],
+      ];
+
+      if ($result = $DB->query($request)) {
```

Comment:
This will fail. `$DB->query()` expects a string.

Reply:
yes, and an other problem fixed with the SUM aggregation